Build failure against PySide >= 6.8

**Environment information**
* Operating System: NixOS
* Cutter version: 
* Obtained from:
  - [x] Built from source
  - [ ] Downloaded release from Cutter website or GitHub 
  - [ ] Distribution repository
* File format:

**Describe the bug**

`PySide::SignalManager::instance()` was removed [here](https://code.qt.io/cgit/pyside/pyside-setup.git/diff/sources/pyside6/libpyside/signalmanager.h?id=33bd61d13d8d9e3794b6049891be62f3351313d9) in [this commit](https://code.qt.io/cgit/pyside/pyside-setup.git/commit/sources/pyside6?id=33bd61d13d8d9e3794b6049891be62f3351313d9), before the release of 6.8.0.

This change in pyside caused a compilation error on `cutter/src/common/PythonManager.cpp:136:28` which called `PySide::SignalManager::instance().clear();`:


```
/build/source/src/common/PythonManager.cpp:136:28: error: 'instance' is not a member of 'PySide::SignalManager'
  136 |     PySide::SignalManager::instance().clear();
```

Inferring from code in that same commit, it seems that this line is no longer needed.
But I'm not sure as I'm not familiar with Qt.

**To Reproduce**

Build cutter against Qt & PySide >= 6.8

**Expected behavior**

No build failure.

**Additional context**

The full compilation log can be viewed on the build server of NixOS. [link](https://hydra.nixos.org/build/274852839/nixlog/1)

--- a/src/common/PythonManager.cpp
+++ b/src/common/PythonManager.cpp
@@ -133,7 +133,9 @@ void PythonManager::shutdown()
     Core()->setProperty("_PySideInvalidatePtr", QVariant());
 
     // see PySide::destroyQCoreApplication()
+#    if QT_VERSION < QT_VERSION_CHECK(6, 8, 0)
     PySide::SignalManager::instance().clear();
+#    endif
     Shiboken::BindingManager &bm = Shiboken::BindingManager::instance();
     SbkObject *pyQApp = bm.retrieveWrapper(QCoreApplication::instance());
     PyTypeObject *pyQObjectType = Shiboken::Conversions::getPythonTypeObject("QObject*");

